packages/relgen-core/src/services/write/remote/github.ts

```diff
@@ -122,15 +126,22 @@ export const githubWriteService = (
           comments,
         });
 
+        const body = [
+          generated.reviews.length > 0
+            ? (generated.summary ?? 'Some notes')
```

Comment:
The phrase 'Some notes' might be too informal or vague for a user-facing string. Consider using a more descriptive phrase like 'Review Summary' or 'Summary of Changes'.